### DISCUSSION TITLE
Not able to run (mvn clean install) the code of microprofile-lra after downloading it

### DISCUSSION THREAD

**Discussion by deepakvatsdv** [eclipse/microprofile-lra#365] [Q&A]
Hi - I am Not able to run (mvn clean install) the code of microprofile-lra after downloading it. I see the below error and not able to fix it. Please help.
<img width="952" alt="image" src="https://github.com/eclipse/microprofile-lra/assets/157034015/3b4db18a-fc2c-4496-92e1-23772ce5c63b">
<img width="952" alt="error" src="https://github.com/eclipse/microprofile-lra/assets/157034015/04e8d114-84b4-4363-a7dd-d6ab3afc24a5">

**Comment by mmusgrov** [comment]
This is a maven version issue. In most of my own projects we use the [Maven Wrapper](https://maven.apache.org/wrapper/) for predictable builds, for example apache-maven-3.8.8 (the first one I picked) builds the project just fine. I think the microprofile-lra project should use the same approach. Do you know if other microprofile projects are using the maven wrapper?

**Reply by mmusgrov** [reply]
@deepakvatsdv I created issue https://github.com/eclipse/microprofile-lra/issues/366 to track this requirement.
